.github/workflows/ci.yml

```diff
@@ -121,3 +121,11 @@ jobs:
         env:
           DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
           DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+
+  cd:
```

Comment:
could you elaborate a bit more?

Reply:
This is how we do it at the open source project I help maintain, we have the flows with the actual logic get invoked from the flows that has the triggers. Here basically we invoke the CD if lint/test/build passes and where the reference comes from a version tag (not 100% sure about this one, since that could be `master` as well, but I don't know if we want it to trigger on each push to master and/or version creations)

Reply:
we want only tags actually to trigger the CD part.